### ISSUE TITLE
Getimagesize results in invalid URL requests for pages with attachments (includes fix)

### ISSUE THREAD

**Issue by c-schmitz** [Kunena/Kunena-Forum#5491]
#### Steps to reproduce the issue
Open a forum page where image attachments are preset on a domain that is using SSL Check  the webserver log, specifically the error log. Not that there are erroneous calls like

`16.4.14.4 - - [27/Jul/2017:14:52:48 +0200] "GET /https://www.domain.org/media/kunena/attachments/31451/Screenshot2013-02-03at1.59.06PM.png HTTP/1.0" 404 162 "-" "-"
`
The correct call would would be

`16.4.14.4 - - [27/Jul/2017:14:52:48 +0200] "GET /media/kunena/attachments/31451/Screenshot2013-02-03at1.59.06PM.png HTTP/1.0" 200 162 "-" "-"
` )
The calls originate from the server itself.

The reason for that erroneous call is in
`/components/com_kunena/template/crypsisb3/layouts/attachment/item/image.php`
at these lines

```
$attachment = $this->attachment;
$location = JUri::root() . $attachment->getUrl();
$data = getimagesize($location);

```
because getUrl already delivers an absolute URL. 

#### Fix
Remove JUri::root() from the second line.

Please incorporate this fix in the next version.
Thank you.

#### System information (as much as possible)

Joomla version: 3.7.4
Kunena version: Kunena 5.0.10
Php version: 5.9.9
Database version: 5.x

**Comment by xillibit** [comment]
> Remove JUri::root() from the second line.

I had added that for fix something in Kunena 5.0.10 or 5.0.9

**Comment by 810** [comment]
Yes, you are correct, i saw it also 2 days ago, i have fixed it already on the k5.1 branch see: https://github.com/Kunena/Kunena-Forum/commit/957aab3d5aade30e4d6e70b87510a9da5cb24396

But i will also fix the image and backporting to k5.0.x
